ansible/roles/postgresql/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# PostgreSQL-related settings
+postgresql_is_local: "{{ 'true' if project_db_host | search( 'localhost|127\\.0\\.0\\.1|^/') else 'false' }}"
```

Comment:
The `search` test was lifted from the `when:` clauses in `ansible/sufia_site.yml`, i.e., it uses the same "local" criteria as used to decide whether to install fedora, solr, or redis.  If we add the IPv6 loopback here, it probably should be added to those, too.

I used 'true' because that seems to be used a lot elsewhere in our roles.  However, after reading your comment, I notice the Python 'True' is also used in a few places, and our usage is inconsistent.

I don't know whether or not Python booleans are more appropriate in Ansible playbooks.  Although Ansible is implemented in Python and uses Jinja2, it is a DSL, not actually Python code.  The Ansible module documentation tends to use `yes` or `no` when describing parameters that take on boolean values, so maybe that is more appropriate?  The [Ansible YAML syntax](https://docs.ansible.com/ansible/YAMLSyntax.html) says that many alternative forms are acceptable for boolean True/False.

The [Jinja2 template documentation on literals](http://jinja.pocoo.org/docs/dev/templates/#literals) actually explicitly prefers `true` and `false` instead of `True` and `False` and indeed states that in previous versions of Jinja2 that `True` and `False` evaluated to and undefined variable (i.e., `false`) but that is no longer the case because of the confusion it caused.  Because the statement line is Jinja2 (because of the {{...}}), arguably `true` and `false` are correct.